geotrigger.js

```diff
@@ -222,7 +222,7 @@
     // create the url for the request
     var url = (geotriggersRequest) ? this.geotriggersUrl + method : method;
 
-    if (this.proxy) {
+    if (this.proxy && !CORS) {
       url = this.proxy + url;
     }
```

Comment:
i definitely understand the workflow of embedding credentials in a proxy so that end users don't have to authenticate, but in this repository the sample proxy thats included doesn't support it and the only mention of proxies is to allow for use in IE8/9.

all that being said, i could also:
* go back to injecting the logic which tests for/utilizes CORS in the editor and faker applications instead
* include a configuration option here so that developers can decide whether or not they want to force use of proxy
* do something else entirely (you'll have to explain what :smile:)